main_test.go

```diff
@@ -18,6 +19,9 @@ func pString(s string) *string {
 	return &s
 }
 
+// Versions of the builders to test
+var generatorVersions = []string{"v0.0.2"}
```

Comment:
how about v1.0.0?

Reply:
we're going to need it before we cut the release

Reply:
If I added it right now, it'll fail because the data isn't there yet